utils/rpms/packaging/Dockerfile.mockbuild

```diff
@@ -32,7 +32,7 @@ RUN dnf -y install mock make                                        \
 # Install bullseye coverage
 ARG JENKINS_URL
 ARG BULLSEYE_KEY
-RUN if [ "x$BULLSEYE" != "x" ]; then                                                \
+RUN set -x; if [ "x$BULLSEYE" != "x" ]; then                                                \
```

Comment:
To be most clear, I think you can remove this entire addition.  But going in steps might be informative.